From: =?utf-8?q?J=C3=B6rg_Frings-F=C3=BCrst?= <debian@jff-webhosting.net>
Date: Thu, 14 Sep 2017 15:28:58 +0300
Subject: m4

---
 configure.in | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/configure.in b/configure.in
index e4df583..bcd28ac 100644
--- a/configure.in
+++ b/configure.in
@@ -1,6 +1,7 @@
 dnl Process this file with autoconf to produce a configure script. -*-sh-*-
 AC_INIT(include/config.h.in)
 AC_CONFIG_HEADER(include/config.h)
+AC_CONFIG_MACRO_DIRS([m4])
 # AC_PREREQ(2.10)dnl		dnl Minimum Autoconf version required.
 # AC_ARG_PROGRAM
 
@@ -21,12 +22,15 @@ XSANE_PACKAGE_VERSION="$PACKAGE-$VERSION"
 AC_DEFINE_UNQUOTED(PACKAGE, "$PACKAGE", [Define to the name of the distribution])
 AC_DEFINE_UNQUOTED(VERSION, "$VERSION", [Define to the version of the distribution])
 AC_DEFINE_UNQUOTED(XSANE_PACKAGE_VERSION, "$XSANE_PACKAGE_VERSION", [Define to name and version of distribution])
+AC_DEFINE([ENABLE_NLS], [], [Description])
+AC_DEFINE([HAVE_CATGETS], [], [Description])
+AC_DEFINE([HAVE_GETTEXT], [], [Description])
+AC_DEFINE([HAVE_LC_MESSAGES], [], [Description])
 AC_SUBST(PACKAGE)
 AC_SUBST(VERSION)
 AC_SUBST(XSANE_PACKAGE_VERSION)
 AC_SUBST(SANE_MAJOR)
 
-
 dnl Check args
 
 echo ""
