From: =?utf-8?q?J=C3=B6rg_Frings-F=C3=BCrst?= <debian@jff-webhosting.net>
Date: Thu, 14 Sep 2017 15:28:58 +0300
Subject: fix_missing_sane-config

---
 configure.in | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/configure.in b/configure.in
index bcd28ac..82393e8 100644
--- a/configure.in
+++ b/configure.in
@@ -165,8 +165,10 @@ fi
 
 AC_SUBST(INTLSUB)
 
-AM_PATH_SANE(1.0.0, HAVE_SANE=yes, )
-
+PKG_CHECK_MODULES([SANE], [sane-backends >= 1.0.0])
+PKG_CHECK_VAR([SANE_PREFIX], [sane-backends >= 1.0.0], [prefix])
+PKG_CHECK_VAR([SANE_LDFLAGS], [sane-backends >= 1.0.0], [ldflags],
+      [HAVE_SANE=yes])
 if test "${USE_GIMP}" = "yes"; then
   if test "${TRY_GIMP_2}" = "yes"; then
     AM_PATH_GIMP_2_0(1.3.23, HAVE_GIMP=yes, )
