From: =?utf-8?q?J=C3=B6rg_Frings-F=C3=BCrst?= <debian@jff-webhosting.net>
Date: Thu, 14 Sep 2017 15:28:58 +0300
Subject: deb_gimp_acquire_menu

---
 src/xsane-text.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/xsane-text.h b/src/xsane-text.h
index 52f7a0f..a1e5956 100644
--- a/src/xsane-text.h
+++ b/src/xsane-text.h
@@ -870,8 +870,8 @@ The format of [DEVICE] is backendname:devicefile (e.g. umax:/dev/scanner).\n\
 #define XSANE_GIMP_INSTALL_HELP		_("This function provides access to scanners and other image acquisition devices through the SANE (Scanner Access Now Easy) interface.")
 
 /* Menu path must not be translated, this is done by the gimp. Only translate the text behind the last "/" */
-#define XSANE_GIMP_MENU_DIALOG		_("<Toolbox>/File/Acquire/XSane: Device dialog...")
-#define XSANE_GIMP_MENU			_("<Toolbox>/File/Acquire/XSane: ")
+#define XSANE_GIMP_MENU_DIALOG		_("<Toolbox>/File/Acquire/XSane/Device dialog...")
+#define XSANE_GIMP_MENU			_("<Toolbox>/File/Acquire/XSane/")
 #define XSANE_GIMP_MENU_DIALOG_OLD	_("<Toolbox>/Xtns/XSane/Device dialog...")
 #define XSANE_GIMP_MENU_OLD		_("<Toolbox>/Xtns/XSane/")
 
