From: =?utf-8?q?J=C3=B6rg_Frings-F=C3=BCrst?= <debian@jff-webhosting.net>
Date: Thu, 14 Sep 2017 15:28:58 +0300
Subject: deb_inhibit_clickthrough

---
 src/xsane.c | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/src/xsane.c b/src/xsane.c
index e7d829c..1e29e80 100644
--- a/src/xsane.c
+++ b/src/xsane.c
@@ -1909,11 +1909,11 @@ void xsane_pref_save(void)
 /* ---------------------------------------------------------------------------------------------------------------------- */
 
 static int xsane_pref_restore(void)
-/* returns true if this is the first time this xsane version is called */
+/* Originally: returns true if this is the first time this xsane version is called */
 {
  char filename[PATH_MAX];
  int fd;
- int result = TRUE;
+ int result = FALSE;
  int i;
 
   DBG(DBG_proc, "xsane_pref_restore\n");
@@ -1927,6 +1927,7 @@ static int xsane_pref_restore(void)
     preferences_restore(fd);
     close(fd);
 
+#if 0
     /* the version test only is done for the local xsane.rc file because each user */
     /* shall accept (or not) the license for xsane */
     if (preferences.xsane_version_str)
@@ -1936,6 +1937,7 @@ static int xsane_pref_restore(void)
         result = FALSE; /* this version already has been started */
       }
     }
+#endif /* 0 */
   }
   else /* no local sane file, look for system file */
   {
@@ -5815,7 +5817,10 @@ static int xsane_init(int argc, char **argv)
     xsane.xsane_rc_set_filename = strdup("xsane"); /* ".rc" is appended later */
   }
 
+  xsane_pref_restore();
+  xsane_pref_save();
 
+#if 0 /* Original XSane code */
   if (xsane_pref_restore()) /* restore preferences, returns TRUE if license is not accpted yet */
   {
     if (xsane_display_eula(1)) /* show license and ask for accept/not accept */
@@ -5828,6 +5833,7 @@ static int xsane_init(int argc, char **argv)
       xsane_pref_save();
     }
   }
+#endif /* Original XSane code */
 
   xsane_pref_restore_media();
 
