From: =?utf-8?q?J=C3=B6rg_Frings-F=C3=BCrst?= <debian@jff-webhosting.net>
Date: Thu, 14 Sep 2017 15:28:58 +0300
Subject: deb_printing_defaults

---
 src/xsane.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/xsane.h b/src/xsane.h
index 67f06d4..e8101e7 100644
--- a/src/xsane.h
+++ b/src/xsane.h
@@ -510,8 +510,8 @@ extern void xsane_batch_scan_add(void);
 
 #define OUT_FILENAME     	"out.pnm"
 #define PRINTERNAME	  	"new printer"
-#define PRINTERCOMMAND  	"lpr"
-#define PRINTERCOPYNUMBEROPTION "-#"
+#define PRINTERCOMMAND  	"lp"
+#define PRINTERCOPYNUMBEROPTION "-n"
 #define FAXPROJECT 	    	"faxproject"
 #define FAXCOMMAND 	 	"sendfax"
 #define FAXRECEIVEROPT		"-d"
