From: =?utf-8?q?J=C3=B6rg_Frings-F=C3=BCrst?= <debian@jff-webhosting.net>
Date: Thu, 14 Sep 2017 15:28:58 +0300
Subject: fix_pdf_xref

---
 src/xsane-multipage-project.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/xsane-multipage-project.c b/src/xsane-multipage-project.c
index f23e5f8..f81eca5 100644
--- a/src/xsane-multipage-project.c
+++ b/src/xsane-multipage-project.c
@@ -973,6 +973,10 @@ static void xsane_multipage_save_file()
     else if (output_format == XSANE_PDF)
     {
       xsane_save_pdf_create_document_header(outfile, &xref, pages, preferences.save_pdf_flatedecoded);
+
+      /* Objects 4 and 5 are unused and do not exist */
+      xref.obj[4] = 0;
+      xref.obj[5] = 0;
     }
   }
 #ifdef HAVE_LIBTIFF
