From: =?utf-8?q?J=C3=B6rg_Frings-F=C3=BCrst?= <debian@jff-webhosting.net>
Date: Thu, 14 Sep 2017 15:28:58 +0300
Subject: fix_preview_mouse_events

---
 src/xsane-preview.c | 9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

diff --git a/src/xsane-preview.c b/src/xsane-preview.c
index cacef95..9416de5 100644
--- a/src/xsane-preview.c
+++ b/src/xsane-preview.c
@@ -80,7 +80,6 @@
 #include "xsane-preview.h"
 #include "xsane-preferences.h"
 #include "xsane-gamma.h"
-#include <gdk/gdkkeysyms.h>
 
 
 #ifndef PATH_MAX
@@ -3023,9 +3022,9 @@ static gint preview_motion_event_handler(GtkWidget *window, GdkEvent *event, gpo
     preview_display_color_components(p, event->motion.x, event->motion.y);
 
     switch (((GdkEventMotion *)event)->state &
-            GDK_Num_Lock & GDK_Caps_Lock & GDK_Shift_Lock & GDK_Scroll_Lock) /* mask all Locks */
+	    (GDK_BUTTON1_MASK | GDK_BUTTON2_MASK | GDK_BUTTON3_MASK)) /* only check for mouse buttons */
     {
-      case 256: /* left button */
+      case GDK_BUTTON1_MASK: /* left button */
 
         DBG(DBG_info2, "left button\n");
 
@@ -3292,8 +3291,8 @@ static gint preview_motion_event_handler(GtkWidget *window, GdkEvent *event, gpo
         }
        break;
 
-      case 512: /* middle button */
-      case 1024: /* right button */
+      case GDK_BUTTON2_MASK: /* middle button */
+      case GDK_BUTTON3_MASK: /* right button */
         DBG(DBG_info2, "middle or right button\n");
 
         if (p->selection_drag)
