From: =?utf-8?q?J=C3=B6rg_Frings-F=C3=BCrst?= <debian@jff-webhosting.net>
Date: Thu, 14 Sep 2017 15:28:58 +0300
Subject: fix_tighten_default_umask

---
 src/xsane.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/xsane.h b/src/xsane.h
index 69fe6bc..32289a0 100644
--- a/src/xsane.h
+++ b/src/xsane.h
@@ -104,7 +104,7 @@
 #define XSANE_DEBUG_ENVIRONMENT	"XSANE_DEBUG"
 
 #define XSANE_PROGRESS_BAR_MIN_DELTA_PERCENT 0.025
-#define XSANE_DEFAULT_UMASK		0007
+#define XSANE_DEFAULT_UMASK		0077
 #define XSANE_HOLD_TIME			200
 #define XSANE_CONTINUOUS_HOLD_TIME	10
 #define XSANE_DEFAULT_DEVICE		"SANE_DEFAULT_DEVICE"
