From: =?utf-8?q?J=C3=B6rg_Frings-F=C3=BCrst?= <debian@jff-webhosting.net>
Date: Thu, 14 Sep 2017 15:28:58 +0300
Subject: man_misleading

---
 doc/xsane.man | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/doc/xsane.man b/doc/xsane.man
index ee363a8..f44fe4a 100644
--- a/doc/xsane.man
+++ b/doc/xsane.man
@@ -203,8 +203,10 @@ When the flag
 or
 .B -N
 is given then xsane reads the next option as default image filename. The name should be
-of the format "name-###.ext". The selection box for filenames is disabled. This
-option normally should be used with the option
+of the format "name-001.ext". (or any other number instead of 001) that 001 part
+will be used as a filename counter if xsane is so configured. Beware that the
+number of digits must correspond to the configuration of xsane. The selection box
+for filenames is disabled. This option normally should be used with the option
 .B --no-mode-selection
 and
 .B --save.
